src/module/Ui/templates/static/de/datenschutz/partials/header.twig

```diff
@@ -0,0 +1,38 @@
+{##
+ # This file is part of Athene2.
+ #
+ # Copyright (c) 2013-2018 Serlo Education e.V.
+ #
+ # Licensed under the Apache License, Version 2.0 (the "License")
+ # you may not use this file except in compliance with the License
+ # You may obtain a copy of the License at
+ #
+ #    http://www.apache.org/licenses/LICENSE-2.0
+ #
+ # Unless required by applicable law or agreed to in writing, software
+ # distributed under the License is distributed on an "AS IS" BASIS
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
+ #
+ # @copyright Copyright (c) 2013-2018 Serlo Education e.V.
+ # @license   http://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ # @link      https://github.com/serlo-org/athene2 for the canonical source repository
+ #}
+{% set archived = revision.isArchived() %}
+{% set date = revision.getDate() %}
+{% set title = 'Datenschutzerklärung' ~ (archived ? ' (archiviert)' : '')  %}
+{% do placeholder('breadcrumbs').set('<li><span>' ~ title ~ '</span></li>') %}
+
+{% if archived %}
+    <div class="alert alert-info" style="margin-top: 20px;">
+        Dies ist eine archivierte Version unserer Datenschutzerklärung. Die aktuelle Version findest du <a href="/datenschutz">hier</a>.
+    </div>
+{% endif %}
+
+{% set prefix = archived ? 'zuletzt geändert am' : 'wirksam ab dem' %}
+{{ pageHeader('Datenschutzerklärung').setSubtitle(prefix ~ ' ' ~ date).render() }}
+
+{% if not archived %}
+    <a href="/datenschutz/archiv">Archivierte Versionen</a>
```

Comment:
This looks a bit strange here, between the two headings.... On the first glance this does look like the page you are viewing is archived, when in fact it's the opposite... Maybe something like "`Archivierte Versionen findest du <a href="/datenschutz/archiv">hier</a>`"...